src/components/Card/Card.tsx

```diff
@@ -72,24 +79,28 @@ export const Card = ({
           // authors={...}
           // meta={...}
         */}
-        <div className="cc-card__meta">
-          {authors && (
-            <dl className="cc-card__authors">
-              <dt className="cc-card__authors-label">Author</dt>
-              {authors?.map(author => (
-                <dd key={author} className="cc-card__author">
-                  {author}
-                </dd>
-              ))}
-            </dl>
-          )}
-          {meta?.date && (
-            <time className="cc-card__date">
-              <FormattedDate dateString={meta?.date} />
-            </time>
-          )}
-        </div>
-        {description && (
+        {isHorizontal && (authors || meta?.date) && (
+          <div className="cc-card__meta">
+            {isHorizontal && authors ? (
+              <dl className="cc-card__authors">
+                <dt className="cc-card__authors-label">Author</dt>
+                {authors?.map(author => (
+                  <dd key={author} className="cc-card__author">
+                    {author}
+                  </dd>
+                ))}
+              </dl>
+            ) : (
+              <span />
```

Comment:
How come the empty span tag?

Reply:
`justify-content: space-between` 😬 

prob should change the styling really

Reply:
Much better now